hack/linter.sh fails on linux/arm64

**Actual behavior**

`make test` fails on arm64 machines because GolangCI-Lint cannot be downloaded for linux/arm64

```
user@debian:~/go/src/github.com/andreasf/kaniko$ uname -a
Linux debian 6.1.0-9-arm64 #1 SMP Debian 6.1.27-1 (2023-05-08) aarch64 GNU/Linux

user@debian:~/go/src/github.com/andreasf/kaniko$ make test
...
RUN /home/user/go/src/github.com/andreasf/kaniko/scripts/../hack/linter.sh
Installing GolangCI-Lint
golangci/golangci-lint crit platform linux/arm64 is not supported.  Make sure this script is up-to-date and file request at https://github.com/golangci/golangci-lint/issues/new
FAILED /home/user/go/src/github.com/andreasf/kaniko/scripts/../hack/linter.sh
make: *** [Makefile:67: test] Error 1
```


**Expected behavior**

`make test` should not fail on linux/arm64

**To Reproduce**
Steps to reproduce the behavior, on an arm64 machine:
1. Make a clean checkout of the repo (without hack/bin directory)
2. Run `make test` on an arm64 machine

--- a/hack/install_golint.sh
+++ b/hack/install_golint.sh
@@ -68,6 +68,7 @@ is_supported_platform() {
     windows/amd64) found=0 ;;
     windows/386) found=0 ;;
     linux/amd64) found=0 ;;
+    linux/arm64) found=0 ;;
     linux/386) found=0 ;;
     linux/ppc64le) found=0 ;;
     linux/s390x) found=0 ;;

